src/pages/WordsOverview.js

```diff
@@ -39,30 +38,24 @@ const WordsOverview = () => {
   return (
     <div className="main_container">
       <h1>Přehled slov</h1>
-
-      <Form />
       <div className="main_container--grid">
         {error && <p>{error}</p>}
         {data.map((oneWord) => {
-          const { id, wordDe, wordCze, sentence } = oneWord;
+          const { id, wordDe, wordCze } = oneWord;
           return (
             <div key={id} className="wordBorder">
```

Comment:
Hmmm, celý tenhle blok má zvlásštní pojmenování tříd. Podívej se na [BEM logic](https://www.youtube.com/watch?v=SLjHSVwXYq4&t=303s) a zkus to podle ní předělat.

Reply:
Předělala jsem - přejmenovala třídy a vytvořila novou komponentu pro přehledovou tabulku = WordsTable component